src/guides/v2.4/graphql/queries/available-stores.md

```diff
@@ -3,14 +3,15 @@ group: graphql
 title: availableStores query
 ---
 
-The `availableStores` query returns configuration information about all store views that have the same parent website. Use this query to implement a store switcher.
+The `availableStores` query returns configuration information about multiple store views. Use this query to implement a store switcher.
 
-{:.bs-callout-tip}
-Specify the [Store header]({{ page.baseurl }}/graphql/send-request.html) to determine the scope of the call.
+If the `useCurrentGroup` input argument is set to `true`, then the `availableStores` query returns configuration information about store views that have the same parent _store_. If the input argument is not specified or is set to `false`, the query returns values for all store views with the same parent _website_.
+
+Specify the [Store header]({{ page.baseurl }}/graphql/send-request.html) to determine the scope of the call. Without this header, the query returns values for the default store view and other store views with the same parent _store_.
```

Comment:
Hi @keharper that's a great explanation. I but the logic under the hood might look a little bit tricky. I would suggest adding something like the following.

> If the store view code specified in the store view header (and `useCurrentGroup` set to `true`), the query will return values for the parent _website_.

It might look like an "extra explanation". But for me, as a developer, will be more clear what should I specify in the store view header. 

Hope in makes sense

Reply:
Please, nevermind. I've reviewed it once again and found this part self-explanatory. 

Thanks.